File: python/1-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        for index, num in enumerate(nums):
            diff = target - num

            if diff in nums:
                diffIndex = nums.index(diff)

                if index != diffIndex:
                    return [index, diffIndex]

File: javascript/1-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = (nums, target, map = new Map()) => {
    for(let i=0; i < nums.length; i++) {
        const diff = target - nums[i];
        if(map.has(diff)) {
            return [map.get(diff), i];
        }

        map.set(nums[i], i);
    }
};